frontends/web/src/components/amount/amount.tsx

```diff
@@ -17,8 +17,8 @@
 import { useContext } from 'react';
 import { AppContext } from '../../contexts/AppContext';
 import { CoinUnit, ConversionUnit } from './../../api/account';
-import { useLocalizedPunctuation } from '../../hooks/localized';
 import style from './amount.module.css';
+import { AmountLocalizationContext } from '../../contexts/amount-localization-context';
```

Comment:
how about naming it just LocalizationContext and maybe in the future we move some other Intl, stuff in there?

Here are a few other Intl calles we have and potentially could go int there one day: 
- https://github.com/digitalbitbox/bitbox-wallet-app/blob/365dfd450cedf2f4e47845516f40ad47c4cefd51/frontends/web/src/components/headerssync/headerssync.tsx#L50
- https://github.com/digitalbitbox/bitbox-wallet-app/blob/365dfd450cedf2f4e47845516f40ad47c4cefd51/frontends/web/src/routes/buy/components/countryselect.tsx#L82
- https://github.com/digitalbitbox/bitbox-wallet-app/blob/365dfd450cedf2f4e47845516f40ad47c4cefd51/frontends/web/src/components/transactions/transaction.tsx#L64
- https://github.com/digitalbitbox/bitbox-wallet-app/blob/365dfd450cedf2f4e47845516f40ad47c4cefd51/frontends/web/src/i18n/utils.ts#L39

Reply:
Yeah, not opposed to naming it Localization.